code/modules/mob/living/carbon/human/species/station/traits_vr/positive.dm

```diff
@@ -2,13 +2,13 @@
 	name = "Haste"
 	desc = "Allows you to move faster on average than baseline."
 	cost = 3
-	var_changes = list("slowdown" = -0.5)
+	var_changes = list("slowdown" = -0.3)
```

Comment:
FYI you cannot have this be anything other than a multiple of the tickrate. It will be rounded to the nearest multiple of the tickrate, which, if you're doing 20fps (0.5 ticklag) is stuff like -0.5 -1.0 etc. Given that -0.5 is Tajara and -1.0 is Teshari... those are the two choices you get basically.